test/common/tls/integration/ssl_integration_test.cc

```diff
@@ -867,6 +893,41 @@ TEST_P(SslCertficateIntegrationTest, ServerEcdsaClientEcdsaOnly) {
   checkStats();
 }
 
+// Server has RSA and ECDSA P-256 certificates, client only supports
```

Comment:
Please add both TLS 1.2 and 1.3 versions of this test. (I'm not sure which version these two tests are, but we need both as they go through slightly different codepaths).

Reply:
As I understand it, this test runs for both TLS 1.2 and 1.3 - at least there are variants for both in the log of the test run

Reply:
Ah, I see now; you're correct. Looks good as-is.